cli/src/pcluster/templates/cdk_builder_utils.py

```diff
@@ -695,6 +695,7 @@ def _build_policy(self) -> List[iam.PolicyStatement]:
             if capacity_reservation_ids:
                 policy.append(
                     iam.PolicyStatement(
+                        sid="AllowRunningReservedCapacity",
```

Comment:
[Minor non blocking] we have two statements related to reserved capacity and both are focused on *running* the capacity:
1. one by reservation id, so the appropriate Sid would be "AllowRunningReservedCapacityByReservationId"
2. another one (line 715) by resource group, so the appropriate Sid would be "AllowRunningReservedCapacityByResourceGroup"

I'm telling this just because "AllowManagingReservedCapacity" does not give the sense of the actual permissions within it.

Reply:
Since this is just a way to run unit tests, I am going to merge the PR and to move the discussion offline. Another PR will be submitted to change sid, if required.